### PR TITLE
Added loadouts: cdf_army_fst/mnt/dst/snw

### PR DESCRIPTION
**Pull request by Freddo3000**

### CODE REVIEW

File: addons/tmf_loadouts/loadouts/cdf_army_template.hpp [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_FieldPack_khk"};
+	backpackItems[] = {
+		LIST_6("CUP_45Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"Binocular"};
+};
```
Comment (AChesheireCat) [inline addons/tmf_loadouts/loadouts/cdf_army_template.hpp:202]: AR and AAR only have a combined ammunition count of 540 which will limit their effectiveness. Consider adding a backpack to AR and giving him more ammunition?

File: addons/tmf_loadouts/loadouts/cdf_army_template.hpp [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+
+class dm : r {
+	displayName = "Designated Marksman";
+	scope[] = {"cup_optic_pso_1_ak_open"};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	vest[] = {DOUBLES(CUP_V_CDF_6B3_3,__CAMO)};
+	backpack[] = {"CUP_B_USMC_AssaultPack"};
```
Comment (AChesheireCat) [inline addons/tmf_loadouts/loadouts/cdf_army_template.hpp:217]: Consider replacing with Kombat backpack (CUP_B_Kombat_Olive) to fit faction? If intended, disregard.

File: addons/tmf_loadouts/loadouts/cdf_army_template.hpp [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+class jp : hp {
+	displayName = "Jet pilot";
+	traits[] = {"engineer"};
+	backpack[] = {"CUP_T10_Parachute_backpack"};
+};
+
+class eng : car {
+	displayName = "Combat Engineer (Explosives)";
+	traits[] = {"engineer", "explosiveSpecialist"};
+	backpack[] = {"CUP_B_USPack_Coyote"};
```
Comment (AChesheireCat) [inline addons/tmf_loadouts/loadouts/cdf_army_template.hpp:491]: Replace with carryall equivalent? Unless intended, then disregard.

**Review by AChesheireCat — CHANGES_REQUESTED** [review]

**Review by BorderKeeper — APPROVED** [review]
from cursory glance I see no issues with the loadouts. If they have been tested locally then lgtm.

**Pull request merged**